setup.py

```diff
@@ -266,9 +266,8 @@ def build_dir():
         "packages": ['khmer', 'khmer.tests', 'oxli', 'khmer._oxli'],
         "package_data": {'khmer/_oxli': ['*.pxd']},
         "package_dir": {'khmer.tests': 'tests'},
-        "install_requires": ['screed >= 1.0', 'bz2file'],
-        "setup_requires": ["pytest-runner>=2.0,<3dev", "setuptools>=18.0",
-                           "Cython>=0.25.2"],
+        "install_requires": ['screed >= 1.0', 'bz2file', 'Cython>=0.25.2'],
```

Comment:
Is cython really needed when you `pip install khmer`? I thought we were doing [this](http://cython.readthedocs.io/en/latest/src/reference/compilation.html#distributing-cython-modules (or planning to) which means you don't need cython to install khmer. This would require the generated .cpp files to be distributed via PyPI and I have no idea how this then works for RTD which doesn't install from PyPI :-/